addons/scopes/functions/fnc_firedEH.sqf

```diff
@@ -24,12 +24,9 @@ private _weaponIndex = [_unit, currentWeapon _unit] call EFUNC(common,getWeaponI
 if (_weaponIndex < 0) exitWith {};
 
 private _adjustment = _unit getVariable [QGVAR(Adjustment), [[0, 0, 0], [0, 0, 0], [0, 0, 0]]];
-private _zeroing = +(_adjustment select _weaponIndex);
+private _zeroing = (_adjustment select _weaponIndex) vectorMultiply 0.05729578; // Convert zeroing from mils to degrees (value of MRAD_TO_DEG(1))
```

Comment:
Would `(_adjustment select _weaponIndex) vectorMultiply MRAD_TO_DEG(1)` really be that much slower?

Reply:
No.